NSS-Day-Cohort-42/rare-API-media-moguls
# Ap comments fixture

#### Changes Made
1. Added file `comments.json` to `fixtures` directory.
2. Modified file `urls.py` to include comments router
3. Added `comment.py` to `views` directory
 - Functionality for get comments by post and get single comment added along with serializers
 - Functionality for delete comments added
​
#### Steps to Review
1. Checkout this branch locally and run the application.
```
git fetch --all
git checkout ap-comments-fixture
```
2. Run the application.
```
npm start
```
3. Open a new Terminal tab (⌘T) and navigate to the server directory.
```
..
server
```
4. Initialize virtual environment, and run the server.
```
pipenv shell
watchgod request_handler.main
```
5. Test app functionality in Postman.
> Create GET request with url http://localhost:8000/comments
> Create GET request with url http://localhost:8000/comments?post_id=1
> Create DELETE request with url http://localhost:8000/comments/6
> Create POST request with url http://localhost:8000/comments and data:
`            "post_id": 2,
            "content": "Postman out here posting",
            "subject": "Posty post",
            "created_on": "2020-11-09T13:19:37+00:00"`

6. View code file.
> Confirm file modifications are present as indicated above.
> Confirm no unused code or extraneous comments exist.
